src/device/mod.rs

```diff
@@ -38,17 +38,17 @@ use std::sync::Arc;
 use std::thread;
 use std::thread::JoinHandle;
 
-use crate::allowed_ips::*;
+use crate::device::allowed_ips::*;
 use crate::crypto::x25519::*;
-use crate::dev_lock::*;
+use crate::device::dev_lock::*;
 use crate::noise::errors::*;
 use crate::noise::handshake::parse_handshake_anon;
 use crate::noise::rate_limiter::RateLimiter;
 use crate::noise::*;
-use crate::peer::*;
-use crate::poll::*;
-use crate::tun::*;
-use crate::udp::*;
+use crate::device::peer::*;
```

Comment:
I think that instead of appending crate you can remove device. i.e. `use peer::*`, `use noise::*`.

Reply:
Okay, I'll refactor it